lib/cluster.js

```diff
@@ -174,20 +176,18 @@ function addListeners() {
 				}
 			}
 		});
+	} else {
+		// Respond to master's requests for worker's local metrics.
+		process.on('message', message => {
```

Comment:
does it still work when moved into this `addListeners` function?

Reply:
Hm yeah you're right. Since that code listens for master's requests of type `GET_METRICS_REQ`, which are only sent when  `clusterMetrics()` is called, I assumed it would be fine, e.g. https://github.com/siimon/prom-client/blob/master/example/cluster.js would work.

However if one is instantiating the `AggregatorRegistry` only for the `master` process, the workers won't have the listener setup. I can fix it by moving the `isWorker` check inside the `process.on` callback though, so that the `cluster` module is not instantiated until an inter-process message is received.